.holo/sources/slate-sbg.toml

```diff
@@ -0,0 +1,3 @@
+[holosource]
+url = "https://github.com/SlateFoundation/slate-sbg"
+ref = "refs/heads/refactor/app-architecture"
```

Comment:
Switch this to master, that's how we want to publish it. When testing locally you can use an env override to build against a feature branch